cicecore/cicedyn/infrastructure/ice_grid.F90

```diff
@@ -616,16 +691,25 @@ subroutine init_grid2
       call ice_timer_stop(timer_bound)
 
       !-----------------------------------------------------------------
-      ! Calculate ANGLET to be compatible with POP ocean model
-      ! First, ensure that -pi <= ANGLE <= pi
+      ! Ensure that -pi <= ANGLE <= pi
       !-----------------------------------------------------------------
 
       out_of_range = .false.
       where (ANGLE < -pi .or. ANGLE > pi) out_of_range = .true.
       if (count(out_of_range) > 0) then
          write(nu_diag,*) subname,' angle = ',minval(ANGLE),maxval(ANGLE),count(out_of_range)
          call abort_ice (subname//' ANGLE out of expected range', &
-             file=__FILE__, line=__LINE__)
+            file=__FILE__, line=__LINE__)
+      endif
+      
+      if (l_readCenter) then
+         out_of_range = .false.
+         where (ANGLET < -pi .or. ANGLET > pi) out_of_range = .true.
```

Comment:
could convert out-of-range values to in-range with cunning use of modulo? Or will this basically never happen?

Reply:
I'm not really sure what this check is protecting against, it may only be bad input data? I just implemented in the same for ANGLET as for ANGLE for consistency.